apicast/src/oauth.lua

```diff
@@ -46,4 +46,13 @@ function _M.call(method, uri, service,...)
   return f, params
 end
 
+function _M.credentials(access_token)
+  if keycloak_configured then
+    oauth = keycloak
```

Comment:
This is using module local variable again. There is no need for it to be module local and can just be local to this function.